android/src/main/java/com/expensify/reactnativekeycommand/KeyCommandModule.java

```diff
@@ -12,6 +12,7 @@
 import com.facebook.react.bridge.WritableMap;
 import com.facebook.react.bridge.WritableNativeMap;
 import com.facebook.react.bridge.ReadableArray;
+import com.facebook.react.bridge.ReadableMap;
 
 import android.view.KeyEvent;
 
```

Comment:
I think we can set import on line 19

Reply:
What do you mean ?

Reply:
Mistyping. 

I think we should remove `Set import` on line 19.

Reply:
Could you elaborate please, why ?

Reply:
I think it is not more used in the file.

Reply:
Oh, yeah. Missed the set Type.